modules/openapi-generator/src/main/resources/ruby-client/configuration.mustache

```diff
@@ -111,9 +111,7 @@ module {{moduleName}}
       @ssl_ca_file = nil
       @ssl_client_cert = nil
       @ssl_client_key = nil
-      @middlewares = []
-      @request_middlewares = []
-      @response_middlewares = []
+      @middlewares = Hash.new { |h, k| h[k] = [] }
```

Comment:
Thanks for the PR. Looks like this is a breaking change, right?

Reply:
I don't think so. I have preserved the existing methods (`use`, `request` and `response`) and modified them to work with this change:  https://github.com/OpenAPITools/openapi-generator/pull/12873/files#diff-dcdc914f4c9bafa7e5d813aedb2fd23e94804ef3a6a6a4fe6d56be7908ddbf75R350

so you can still configure middlewares by calling the existing methods:

```rb
  config.use :instrumentation, name: "request.faraday.petstore"
```